source/includes/_tipser-widget.md

```diff
@@ -105,6 +105,26 @@ tipser.elements('myPosId')
 It is possible to combine multiple invocations of `mountContent()` with zero or one invocations of `mountCart()` as presented in the code snippet. That will lead to multiple pieces of content
 AND a cart icon being displayed on the page.
 
+## Advanced example: Custom checkout popup ##
+
+Default Checkout Popup appears upon adding product to the cart. It can be however overridden by fully custom popup via `customCheckoutPopupHandler` configuration parameter. See snippet with an example:  
+
+```js
+tipser.elements('myPosId', {
+    customCheckoutPopupHandler: {
+        init: function (openPurchaseDialog){
```

Comment:
Do we really need two steps here? No way to do it with just one function?

Reply:
I was thinking about that in the first place and I got:

```
customCheckoutPopupHandler: function(product, openPurchaseDialog) {
    const checkoutPopup = document.querySelector("#my-custom-checkout-popup");
    checkoutPopup.querySelector('.title').textContent = product.title;
    checkoutPopup.querySelector('.price').textContent = product.priceIncVat.formatted;
    checkoutPopup.querySelector('.checkout-button').addEventListener('click',openPurchaseDialog);
}
```

The Checkout Popup may be implemented in two ways on client side:
1. html element is reused between adding two different products
2. each time product is added to cart, new html markup for checkout popup is created

Above implementation is okay for 1st case, but for 2nd we would end up with tricky logic to avoid adding multiple event listeners to `button`.

As both cases may happen (it's hard to predict for us, as we don't control that flow anymore), I found it important to make life easier in case client is case 2nd.

What do you think ? 

I have experience with vanilla js / jquery app building, and I know that both cases may happen, thus I found important to make life easier for devs